### PR TITLE
getConsumerReservations with date is optional param

### PR DESCRIPTION
**Pull request by bhumisha**

### CODE REVIEW

**Comment by bhumisha** [comment]
without Date Parameter
Screen Shot 2021-05-18 at 3 23 22 PM

With Parameter
Screen Shot 2021-05-18 at 3 23 49 PM

File: server/schemas/resolvers.js [hunk 1]
```diff
@@ -63,26 +63,32 @@ const resolvers = {
...
-    //         match: { "provider" : context.user._id}
-    //      })
-    //     return reservedParkingPlaces
-    //   }
-
-    //   throw new AuthenticationError("No logged in user found");
-    // },
+    //Get Consumers current and future resesrvation based on date criteria.
+    getConsumerReservations: async (parent, args, context) => {
+      const { startDate } = args;
```
Comment (yulduzetta) [inline server/schemas/resolvers.js:68]: typo? should be startDate

Comment (bhumisha) [reply server/schemas/resolvers.js:68]: where? startDate

Comment (yulduzetta) [reply server/schemas/resolvers.js:68]: my bad, 😬

**Review by yulduzetta — APPROVED** [review]
ship it!

**Pull request merged**